docs/sources/registry.rst

```diff
@@ -0,0 +1,39 @@
+Source: registry
+==============
+
+The ``registry`` push source allows push container content from external docker
+registry
+
+Supported content types:
+
+* Container images
+
+
+registry URLs
+------------------
+
+Registry source can be obtained by specifying URL:
+
+``registry:registry-hostname/namespace/repository:tag``
```

Comment:
I don't think this is accurate for the current implementation.  I also can't find any test which covers this (`Source.get` with a `registry:` URL).

If you want to make it work as documented here, you probably have to rename `registry_uris` to just `url` and/or do some other tricks.

How do you want it to work in the case of more than one image?  If you want e.g. this to work:

`registry:registry.access.redhat.com/ubi7/ubi:latest,registry.access.redhat.com/ubi8/ubi:latest`

... and to produce push items for both the ubi7 and ubi8 images, which seems reasonable to me, I would strongly suggest writing a test for it because I don't think it works right now.